File: engine/model/src/main/scala/tools/packet/message/YellowTip.scala
package tools.packet.message

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class YellowTip(private var _tip: String) extends PacketInput {
  def tip: String = _tip

  override def opcode(): SendOpcode = SendOpcode.SET_WEEK_EVENT_MESSAGE
}

File: engine/engine-model/src/main/java/tools/packet/message/YellowTip.java
package tools.packet.message;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record YellowTip(String tip) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.SET_WEEK_EVENT_MESSAGE;
   }
}